src/main/java/org/kainos/ea/cli/JobRequest.java

```diff
@@ -0,0 +1,65 @@
+package org.kainos.ea.cli;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobRequest {
```

Comment:
you don't need a job request object as you're not requesting any data on this endpoint